### PR TITLE
Update user_channel.rb

### PR DESCRIPTION
**Pull request by melwhite1**
Fixes #0000 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

**Comment by melwhite1** [comment]
Hey there! My change needs review!

File: app/channels/user_channel.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
   def subscribed
-    if !current_user.nil?
-      stream_from "users:#{current_user.id}"
-    else
-      reject
-    end
-  end
++    if current_user.nil?
+       reject
++    else
```
Comment (cesswairimu) [inline app/channels/user_channel.rb:5]: Hi @melwhite1, thanks for opening this. The `+` should not be part of the code change. could you please check?. Also please update the pull request description with the issue number where you have `Fixes #0000`, It helps associate the PR to the issue and helps in closing once its merged. Thanks

**Review by cesswairimu — CHANGES_REQUESTED** [review]

File: app/channels/user_channel.rb (hunk 1 above)
Comment (melwhite1) [reply app/channels/user_channel.rb:5]: Hi! Ok, I think I fixed it!

Comment (cesswairimu) [reply app/channels/user_channel.rb:5]: Hi @melwhite, I saw the new pull request and it is a little amiss. This looks closer than the other one, you could try removing the `+` at the beginning of line 3,5 and 6. If you can edit from here https://github.com/melwhite1/plots2/edit/patch-1/app/channels/user_channel.rb and commit this pull request should have the new changes. What do you think? Thanks for sticking with it

Comment (cesswairimu) [reply app/channels/user_channel.rb:5]: HI @melwhite1, so you need any help here? Its so close and we are happy to help if you are stuck. Thanks

**Comment by cesswairimu** [comment]
This PR has been open for a long while and no activity/ reviews requested has not been addressed. We understand you might be busy and engaged on other things. I am closing this for now and reassigning the FTO to someone else. Feel free to come back anytime and request a new FTO. Thanks

**Pull request closed without merging**